src/processor/notification.rs

```diff
@@ -73,6 +75,8 @@ pub enum Notification {
     ReplayLog(IdentifierPrefix),
     #[cfg(feature = "query")]
     ReplyKsn(SignedReply),
+    #[cfg(feature = "query")]
+    GetMailbox(QueryArgsMbx),
```

Comment:
The `GetMailbox` shall not be used for querying. If there's a need to query smth, especially via `qry` msg, it is sufficient to impl it in a way that the incoming `qry` is being processed (validated) by event processor and then reply is generated. 

Right now event processor returns `Notification`, but it'd be simpler to have it separated. So that all the validation that takes place in query processor is extracted. So that when messages like `qry` appear, they can be just validated against proper integrity. On the other hand, when messsages like `icp` or `rot` appear, they continue to use event processor. Then replying for `qry` is much simpler as it doesn't involve any event processing logic which by default assumes that any event entering this space will change the state (will be stored in DB for example).